huxflux/huxflux.js

```diff
@@ -1,10 +1,28 @@
+
 /**
- * @returns {Number[]} Lista med sju nummer
+ * @returns {Number[]} list of seven integers
  */
 function huxflux() {
-    return; // Returnera din lista med nummer här
+
+    const Number = []
+   
+    while(Number.length<7){
+  
+      do{  //the do while loop will fill the Number array with random integers. 
+        var occuring = false  
+        var random = Math.floor(Math.random() * 35) + 1
+        Number.forEach(number=>{  //loops through the array and compares the random integer with all the integers that are in the array
+          if(number===random) occuring = true  
```

Comment:
Nice use of comparison and logical operator in line 15, as it is best practice to use '===" rather than '==', the first one compares the value and type of data 👍🏻